numpy/core/src/umath/ufunc_object.c

```diff
@@ -1021,6 +1071,123 @@ _borrowed_reference(PyObject *obj, PyObject **out)
     return NPY_SUCCEED;
 }
 
+
+static int
+_parse_signature_obj(PyUFuncObject *ufunc,
```

Comment:
It looks like this just moved. If you're going to move existing functions around, can you do it in a separate self-contained PR so that it's easy to review?

Reply:
Nevermind, this didn't move.

Reply:
Yes, will look it into (later), sorry about that. The `use_min_scalar_type` is also just moved around, I think the rest should be close to where it was.